### ISSUE TITLE
Configuration howtos/examples

### ISSUE THREAD

**Issue by immanuel-h** [joe-akeem/contour-tiles#17]
Hi, it would be great if we could get some examples on how to configure this container based on our needs. For example ...

* How to set the line intervals? Right now this seems to be hardcoded to 20.
* How to set the min/max zoom? Right now 20m lines just disappear at a fixed zoom level
* Is there any way to smooth/interpolate the generated lines? I.e. they are pretty jagged right now, and a trivial interpolation step might be helpful to make them look much more natural

**Comment by joe-akeem** [comment]
Yes, intervals are currently hardcoded (`-i` parameter of `gdal_contour`, I think). Should be easy to make this configurable. Intend to look into your ideas along with some of the requests soon but can't promise anything...
